### DISCUSSION TITLE
Html banner

### DISCUSSION THREAD

**Discussion by smolf** [tltneon/lgsl#120] [General]
Hi LGSL :)
a question, how to get html banner, banner does not work on my website.

Thank you again.

**Comment by BaxAndrei** [comment]
Do you have all requirements with "yes" on admin panel? (GD should be "yes" to be able to use banners)

**Reply by smolf** [reply]
Hi again.
Det link in banners, is not html to use on website?
![test](https://user-images.githubusercontent.com/5747408/177821136-0956f66d-68bf-4b5d-8e57-91e8482bc51d.png)
![test2](https://user-images.githubusercontent.com/5747408/177821148-1ccdf99d-760a-4bc3-84fb-eacbd250bf0a.png)
n

**Reply by tltneon** [reply]
@smolf Hi!

If you need html, you should transform by yourself for now cause it feature not implemented yet. It will looks like `<a href="details_page_link"><img src="banner_link" /></a>`

**Reply by smolf** [reply]
Thank you so much ;)
